### ISSUE TITLE
 Error: Requiring unknown module "undefined"

### ISSUE THREAD

**Issue by gveraperea** [gusgard/react-native-swiper-flatlist#169]
React Native version: 0.71.7 (not expo)

## Steps To Reproduce

1. Install the library on version ^3.1.0
2. Try example code or any code. Error happens when trying to use `SwiperFlatList`

```
import React from 'react';
import {Text, Dimensions, StyleSheet, View} from 'react-native';
import {SwiperFlatList} from 'react-native-swiper-flatlist';

const colors = ['tomato', 'thistle', 'skyblue', 'teal'];
export const TestCarousel = () => {
  return (
    <View style={styles.container}>
      <SwiperFlatList
        autoplay
        autoplayDelay={2}
        autoplayLoop
        index={2}
        showPagination
        data={colors}
        renderItem={({item}) => (
          <View style={[styles.child, {backgroundColor: item}]}>
            <Text style={styles.text}>{item}</Text>
          </View>
        )}
      />
    </View>
  );
};
const {width} = Dimensions.get('window');
const styles = StyleSheet.create({
  container: {flex: 1, backgroundColor: 'white'},
  child: {width, justifyContent: 'center'},
  text: {fontSize: width * 0.5, textAlign: 'center'},
});

```

Screenshot:
![Screenshot 2023-05-11 at 18 06 57](https://github.com/gusgard/react-native-swiper-flatlist/assets/53072907/bbd247f4-bdff-48bf-b9ee-21fb01a4ce5e)

![Screenshot 2023-05-11 at 18 08 49](https://github.com/gusgard/react-native-swiper-flatlist/assets/53072907/680f3250-8cc3-4bf2-9fc0-c3cedf7e48a3)

**Comment by koloboid** [comment]
@gveraperea 
I'd catch the same error. Found this [line](https://github.com/gusgard/react-native-swiper-flatlist/blob/master/src/components/SwiperFlatList/SwiperFlatList.tsx#L267
), just installed react-native-gesture-handler and the error has gone away.

 @gusgard maybe we should add this dep to peerDependency? It seems Metro is stricter now with such requires.
(thanks for the package btw!)

**Comment by gusgard** [comment]
Fixed in `3.2.3`, if you need to use `react-native-gesture-handler`, use the following import `import { SwiperFlatListWithGestureHandler } from 'react-native-swiper-flatlist/WithGestureHandler';`
